source/common/http/conn_manager_impl.cc

```diff
@@ -1498,6 +1498,7 @@ void ConnectionManagerImpl::ActiveStreamFilterBase::commonContinue() {
                    static_cast<const void*>(this));
   ASSERT(stopped_);
   stopped_ = false;
+  stopped_all_ = false;
```

Comment:
nit: Instead of `stopped_` and `_stopped_all_` can we make this a tri-state enum? I think it will help the logic be more clear.

Reply:
Fixed. Thanks!